op-e2e/e2eutils/geth/wait.go

```diff
@@ -131,8 +127,6 @@ func WaitForBlockTag(number *big.Int, client *ethclient.Client, timeout time.Dur
 			if block.NumberU64() >= number.Uint64() {
 				return client.BlockByNumber(ctx, number)
 			}
-		case <-timeoutCh:
```

Comment:
Also here, we can remove the `timeoutCh`, but I think we should still also select on `<-ctx.Done()`. We should also check the error returned by `block, _ := client.BlockByNumber(ctx, big.NewInt(tag))`, it is currently thrown away.